smallsetup/UninstallK8s.ps1

```diff
@@ -108,11 +108,8 @@ if ($global:PurgeOnUninstall) {
     Remove-Item -Path "$global:NssmInstallDirectory\nssm.exe" -Force -ErrorAction SilentlyContinue
     Remove-Item -Path "$global:SetupJsonFile" -Force -ErrorAction SilentlyContinue
     Remove-Item -Path "$global:KubernetesImagesJson" -Force -ErrorAction SilentlyContinue
-    Remove-Item -Path "$global:BinPath\kube*.exe" -Force -ErrorAction SilentlyContinue
     Remove-Item -Path "$global:BinPath\cri*.exe" -Force -ErrorAction SilentlyContinue
-    Remove-Item -Path "$global:BinPath\vc_redist*.exe" -Force -ErrorAction SilentlyContinue
```

Comment:
@abhitoshj  these removal implies kube tools are cleaned as part of ExecutableFolderPath correct?

Reply:
Also, for upgrade scenario, we should cleanup the old instance during the upgrade.

Reply:
kubetools are deleted as part of ExecutableFolderPath

For upgrade scenario, I can bring back the removal from bin folder for kubectl. vc_redist and helm are not being downloaded so probably their removal can remain unchanged.

Some points to note here:
- k2s does not have yet an old version. Should the upgrade scenario be handled now ?
- When the old version of k2s is uninstalled, the bin folder will be removed from PATH. At the same time, the new version always uses kubectl from its own bin directory, so even if kubectl is not deleted from old version's bin, it does not cause hinderance. 
- After the upgrade is finished, the user will be asked to reload shell so the old version's kubectl can never be resolved from PATH
- If we bring back this code, then at some point of time, we should also plan to remove this because it will not be applicable.